### PR TITLE
BlinkrFixes - Fixed broken links

### PR DESCRIPTION
**Pull request by Dantheman720**

### CODE REVIEW

**Comment by ollyjshaw** [comment]
retest this please

File: products/eap/docs-and-apis.adoc [hunk 1]
```diff
@@ -2,11 +2,6 @@
 
 == Extra Configurations
 
-=== More Resources
```
Comment (paulrobinson) [inline products/eap/docs-and-apis.adoc]: I'm fixing this in another PR. You can revert this change here.

File: _config/blinkr.yaml [hunk 2]
```diff
@@ -30,6 +30,15 @@ skips:
...
   - !ruby/regexp /^https:\/\/docs.jboss.org\/author\/display\/PLINK\/SAML\+Authenticators\+\%28Tomcat\%2CJBossAS\%29+/
+#   Remove these before merge
+  - !ruby/regexp /^http:\/\/it-developers-pr.stage.redhat.com\/pr\/\d{3}\/build\/\d{3}\/video\/youtube\/6wgUTUzjH90/
+  - !ruby/regexp /^http:\/\/it-developers-pr.stage.redhat.com\/pr\/\d{3}\/build\/\d{3}\/video\/youtube\/oD9mroqMGUk/
+  - !ruby/regexp /^http:\/\/it-developers-pr.stage.redhat.com\/pr\/\d{3}\/build\/\d{3}\/video\/youtube\/fSVIGEdJjHs/
+  - !ruby/regexp /^http:\/\/it-developers-pr.stage.redhat.com\/pr\/\d{3}\/build\/\d{3}\/video\/youtube\/HpW_PzQlzrg/
+  - !ruby/regexp /^http:\/\/it-developers-pr.stage.redhat.com\/pr\/\d{3}\/build\/\d{3}\/video\/youtube\/a4YTLbI-UfI/
+  - !ruby/regexp /^http:\/\/it-developers-pr.stage.redhat.com\/pr\/\d{3}\/build\/\d{3}\/video\/youtube\/g9nDUmyzsuU/
+  - !ruby/regexp /^http:\/\/it-developers-pr.stage.redhat.com\/pr\/\d{3}\/build\/\d{3}\/video\/youtube\/J7XGqPaSntI/
+  - !ruby/regexp /^http:\/\/it-developers-pr.stage.redhat.com\/pr\/\d{3}\/build\/\d{3}\/video\/youtube\/5_qsqPOi6kE/
```
Comment (paulrobinson) [inline _config/blinkr.yaml]: I think these are covered by the DCP cleanups I did. Can you revert this?

Comment (Dantheman720) [inline _config/blinkr.yaml]: Yep.

File: _config/blinkr.yaml [hunk 3]
```diff
@@ -151,6 +152,7 @@ ignores:
   - url: https://infinispan.ci.cloudbees.com/ #ISPN-4600
   - url: http://fisheye.jboss.org/browse/Guvnor #GUVNOR-2113
   - url: https://jboss.org/jbossas/osgi #JBAS-9573
+  - url: http://www.redhat.com/developers/rhel/RHEL_Developer_Getting_Started_Guide.pdf #RHD-1342
```
Comment (paulrobinson) [inline _config/blinkr.yaml]: This link can now be fixed. use: http://static.jboss.org/rhd/docs/rhel_developer_getting_started_guide.pdf

**Comment by paulrobinson** [comment]
needs a rebase now :-(

**Comment by paulrobinson** [comment]
Otherwise, ready to merge

**Comment by Dantheman720** [comment]
retest this please

**Comment by paulrobinson** [comment]
Looks like we are out of disk space. I've asked @ollyjshaw to investigate.

**Comment by Dantheman720** [comment]
retest this please

File: products/brms/get-started.adoc [hunk 4]
```diff
@@ -158,9 +158,9 @@ http://www.schabell.org/2015/07/workshop-in-can-jboss-brms-full-day-workshop.htm
 Below you will find links to all the materials, articles, blogs, video channels, demo collections and other content that you might want to bookmark as your JBoss BRMS collection.
 
 * link:#{site.base_url}/products/brms/index.html[JBoss BRMS product page]
-* link:#{site.base_url}/products/brms/developer-materials/#!projectCode=brms[Developer materials for JBoss BRMS]
+* link:#{site.base_url}/resources/#!project=brms[Developer materials for JBoss BRMS]
```
Comment (paulrobinson) [inline products/brms/get-started.adoc]: This should stay in the product pages. So link to the learn page for BRMS. E.g:

      link:#{site.base_url}/products/brms/learn/#!projectCode=brms[Developer materials for JBoss BRMS]

File: products/brms/get-started.adoc [hunk 5]
```diff
@@ -158,9 +158,9 @@ http://www.schabell.org/2015/07/workshop-in-can-jboss-brms-full-day-workshop.htm
...
 * http://www.schabell.org/search/label/Tips%26Tricks[JBoss BRMS tips and tricks]
-* link:#{site.base_url}/products/brms/developer-materials/#!formats=video&projectCode=brms[JBoss BRMS videos]
+* link:#{site.base_url}/resources/#!formats=video&project=brms[JBoss BRMS videos]
```
Comment (paulrobinson) [inline products/brms/get-started.adoc]: same here

**Comment by paulrobinson** [comment]
Merging, as the incorrect links are better than 404s. I'll raise another PR to fix them.
